service/src/test/java/uk/gov/hmcts/reform/fpl/service/document/DocumentsListRendererTest.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.reform.fpl.service.document;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.model.documentview.DocumentBundleView;
+import uk.gov.hmcts.reform.fpl.model.documentview.DocumentView;
+import uk.gov.hmcts.reform.fpl.service.CaseUrlService;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.utils.ResourceReader.readString;
+
+@ExtendWith(MockitoExtension.class)
+class DocumentsListRendererTest {
+
+    private static final String IMAGE_BASE_URL
+        = "https://raw.githubusercontent.com/hmcts/fpl-ccd-configuration/master/resources/";
+
+    private final CaseUrlService caseUrlService = mock(CaseUrlService.class);
+
+    private final DocumentsListRenderer underTest = new DocumentsListRenderer(IMAGE_BASE_URL, caseUrlService);
+
+    @Test
+    void shouldRenderDocumentBundles() {
+        List<DocumentBundleView> documentBundleViews = List.of(
+            DocumentBundleView.builder()
+                .name("Applicant's statements and application documents")
```

Comment:
minor: could improve later: all the decision paths are covered, but they both included in 1 test only. Ideally there should be 3 separate tests, full/empty and multiple elements

Reply:
missing coverage for uploadedAt/uploadedBy